src/core_plugins/metrics/public/components/panel_config/table.js

```diff
@@ -77,6 +92,56 @@ class TablePanelConfig extends Component {
                   value={model.pivot_rows}
                 />
               </div>
+              <div className="vis_editor__vis_config-row">
```

Comment:
Is there a reason why this can't reuse the `DataConfig` component used in metric and gauge mode?

Reply:
This just got missed when I created DataConfig to reduce redundant code

Reply:
This is fixed